pkgs/top-level/all-packages.nix

```diff
@@ -11606,6 +11606,8 @@ with pkgs;
 
   qscintilla-qt4 = callPackage ../development/libraries/qscintilla-qt4 { };
 
+  qsv = callPackage ../tools/text/qsv { };
```

Comment:
The letter `s` comes after the letter `r`. Please add this entry at the right place.